tests/__init__.py

```diff
@@ -96,6 +96,7 @@ def test_smallest(self):
         commands = (
             # Must generate a configuration file
             ("test", "-f", "/opt/odoo/auto/odoo.conf"),
+            ("test", "-d", "/opt/odoo/custom/src/private"),
```

Comment:
Please add also `test -d custom/ssh`, and remove the `ssh/.empty` file (to ensure that it will exist even if your current scaffolding has no `ssh` dir)